src/link/internal.tsx

```diff
@@ -98,16 +98,16 @@ const InternalLink = React.forwardRef(
       <>
         {children}
         {external && (
-          <>
-            {' '}
+          <span className={styles['icon-wrapper']}>
+            &nbsp;
             <span
               className={styles.icon}
               aria-label={externalIconAriaLabel}
```

Comment:
Do you think this label and role should move on the wrapping span to avoid an announcement "non breaking space, opens in a new tab"? (not sure if this is really happening, just asking)

Reply:
Fine with me too